src/index.ts

```diff
@@ -318,27 +314,6 @@ class ExtensibilitySdk {
     });
   };
 
-  /**
-   * Sends request to Outreach hosting app to display the configuration form.
-   *
-   * @memberof ExtensibilitySdk
-   */
-  public configure = async () => {
```

Comment:
Is it safe to delete all those methods and parameters? Shall we maybe just mark it deprecated and remove later?

Reply:
Configure is special case - original idea was that user starting 1st time add-on will provide its own config - so this was supposed to trigger config screen on flagship.

We are definitely not doing that at all and never will - instead we ask for config values during the configuration and that flow has nothing to do with this.

I am 99.99% sure no add-on creator is using this method - its purpose was to demo concepts 2 years ago